mapper/orm-coordination-outbox-polling/src/test/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/event/impl/EventProcessorClusterLinkMixedSharding4ShardSelfDynamicBaseTest.java

```diff
@@ -28,18 +30,24 @@ abstract static class AbstractBaseTest extends AbstractEventProcessorClusterLink
 		// to demonstrate that the ID order doesn't matter when using static sharding:
 
 		@Override
-		protected long other1Id() {
-			return AbstractEventProcessorClusterLinkTest.SELF_ID + 1;
+		protected UUID other1Id() {
+			return UUID.fromString( AbstractEventProcessorClusterLinkTest.SELF_ID.toString()
+					.replace( "000000000042", "000000000043" )
```

Comment:
This is getting quite confusing...

Can't you you add a static method `toUUID(long)` and use that everywhere? That way you can keep `Long` constants and (relatively) clear ordering here.

You'll probably have to duplicate the constant `SELF_ID` into `SELF_ID_ORDINAL` (long) + `SELF_ID` (UUID) to avoid changing to many pieces of code.